tendrl-api.spec

```diff
@@ -78,10 +111,70 @@ install -Dm 0644 README.adoc Rakefile $RPM_BUILD_ROOT%{_datadir}/doc/tendrl
 install -Dm 0644 config/apache.vhost.sample $RPM_BUILD_ROOT%{_sysconfdir}/httpd/conf.d/tendrl.conf
 install -Dm 0644 config/*.* $RPM_BUILD_ROOT%{_datadir}/doc/tendrl/config/
 
+# Install SELinux interfaces and policy modules
+install -d %{buildroot}%{_datadir}/selinux/devel/include/%{moduletype}
+install -d %{buildroot}%{_datadir}/selinux/packages
+
+# tendrl
+install -p -m 644 selinux/tendrl.if \
+        %{buildroot}%{_datadir}/selinux/devel/include/%{moduletype}
+install -m 0644 selinux/tendrl.pp.bz2 \
+	%{buildroot}%{_datadir}/selinux/packages
+
+# carbon
+install -p -m 644 selinux/carbon.if \
+        %{buildroot}%{_datadir}/selinux/devel/include/%{moduletype}
+install -m 0644 selinux/carbon.pp.bz2 \
+        %{buildroot}%{_datadir}/selinux/packages
+
+%post -n tendrl-server-selinux
+%_format MODULE %{_datadir}/selinux/packages/tendrl.pp.bz2
```

Comment:
We have SELinux policy macros for this changes in post install phase. For more info see: https://fedoraproject.org/wiki/SELinux/IndependentPolicy#The_.25post_Section

Reply:
Sure, Thank you, i will use those macros